README.md

```diff
@@ -1,2 +1,166 @@
-# capstone-IOT
-A website to bring awareness to IOT development
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖IOT program Page <a name="about-project"></a>
+
+> I built a webpage for an IOT program, using data from some journals and a design of [Cindy Shin](https://www.behance.net/adagio07).
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The site was built with HTML , CSS and Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **The home page displays all the main informatio  about the program**
+- **The about page displays information about previous programs**
+- **It has a page for mobile version**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live App Link](https://danutibe07.github.io/capstone-IOT/)
+- [Loom video link](https://www.loom.com/share/47d3efa1cb414d59bd466293d52d3d3d)
```

Comment:
- [ OPTIONAL ] I think it would be nice if you highlight some interesting piece of code or something you built that you are very proud of. since it is very important to show that you learned something new or somthing you are proud to build it in the presentation video 🤩🚀💯🙏🏻🙏🏻